src/components/dialogs/addProject.tsx

```
@@ -511,11 +521,8 @@ export const AddProject: React.FC<AddProjectProps> = ({ onFormSubmit }) => {
                     <FormItem>
                       <FormLabel>Live Demo Link</FormLabel>
                       <FormControl>
-                        <Input placeholder="Enter live demo link" {...field} />
+                        <Input placeholder="Enter live demo link (optional)" {...field} />
```

ESLint

[error] Replace `·placeholder="Enter·live·demo·link·(optional)"·{...field}` with `⏎··························placeholder="Enter·live·demo·link·(optional)"⏎··························{...field}⏎·······················`

(prettier/prettier)